mc-oblivious-ram/src/evictor/mod.rs

```diff
@@ -23,11 +27,16 @@ use aligned_cmov::typenum::PartialDiv;
 
 use aligned_cmov::ArrayLength;
 
+use aligned_cmov::CMov;
+
 use rand_core::CryptoRng;
 
-use crate::path_oram::{BranchCheckout, MetaSize};
+use crate::path_oram::{meta_is_vacant, meta_leaf_num, BranchCheckout, MetaSize};
 use rand_core::RngCore;
 
+use core::convert::TryFrom;
+const FLOOR_INDEX: usize = usize::MAX;
```

Comment:
there should be a code comment on this const explaining what it's significance is